app/assets/stylesheets/views/_worldwide-organisation.scss

```diff
@@ -1,10 +1,51 @@
+// stylelint-disable max-nesting-depth
 .worldwide-organisation-header {
```

Comment:
This is a continuation of what was there before but as you are adding more to it perhaps it would be best if this was written to be more in-line with [guidance in the GDS Way](https://gds-way.cloudapps.digital/manuals/programming-languages/css.html#sass-nesting). Personally I think 'show-other-content', 'logo' and 'metadata' could be moved from out of worldwide-organisation-header and the BEM naming convention could be used here (worldwide-organisation-header__logo for instance).

Reply:
Great,

I've made this change as a separate commit. I've left the `show-other-content` class as-is because it's related to JS, but I've nested it in a `js-enabled` so it's clear.